test/stat_tracker_test.rb

```diff
@@ -32,6 +32,77 @@ def test_find_team
     assert_equal "Houston Dynamo", @stat_tracker.find_team_name("3")
   end
 
+  # Game Stats
```

Comment:
Thank you for adding these!! We will have to adjust the expected values to match our new dummy files :)